.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--- Prepend PR title with [WIP] if work in progress. Remove when ready for review. -->
+
+<!--- For a timely review/response, please avoid force-pushing additional commits if your PR already received reviews or comments -->
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Documentation Update
+
+## Description
+
+## Related Tickets & Documents
+
+## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
+
+## Added to documentation?
+
+- [ ] docs.dev.to
```

Comment:
This reminds me we need to update/replace docs.codebuddies.org (which refers back to github.com/codebuddies/codebuddies)... but in the meantime, could we remove this `Added to documentation` section? Thanks!